### DISCUSSION TITLE
[navigation] Cursor to top/bottom of current view

### DISCUSSION THREAD

**Discussion by seanbone** [helix-editor/helix#7274] [Q&A] [answered]
I'm coming from vim keybindings, where I often use H and L to jump to the top/bottom of the current view (there is also M for middle). It is so far the most effective way I've found of navigating vertically I've found, or at least, one that works for me. When I know I need to edit something that is just off the top/bottom of the view, I don't like to have to think of something to search for with `/`.

After searching through the [keymaps](https://docs.helix-editor.com/keymap.html) and the command palette, I've not found anything that emulates this kind of motion in Helix. 

Is there a direct equivalent I haven't found? Or does Helix have other mechanisms for vertical navigation I'm not aware of?

**Comment by the-mikedavis** [comment, marked as answer]
I believe you're looking for `gt` "goto top", `gb` "goto bottom" and `gc` "goto center" https://docs.helix-editor.com/master/keymap.html#goto-mode

**Reply by seanbone** [reply]
That is exactly it. Thank you!

I'm really enjoying Helix so far and hope it can cure me of my "evil" ways :D
